rcsb/utils/dictionary/DictMethodEntryHelper.py

```diff
@@ -621,18 +623,22 @@ def consolidateAccessionDetails(self, dataContainer, catName, **kwargs):
         try:
             logger.debug("Starting with  %r %r %r", dataContainer.getName(), catName, kwargs)
             #
-            # Add missing pdbx_database_status for MA models
+            # Add missing pdbx_database_status for MA or AF models (if absent in mmCIF file)
             cName = "pdbx_database_status"
-            if dataContainer.exists("entry"):
-                dObj = dataContainer.getObj("entry")
-                entryId = dObj.getValue("id", 0)
-                if entryId.startswith("ma"):
-                    if not dataContainer.exists(cName):
-                        dataContainer.append(DataCategory(cName, attributeNameList=self.__dApi.getAttributeNameList(cName)))
-                        eObj = dataContainer.getObj(cName)
-                        eObj.setValue(entryId, "entry_id", 0)
-                        eObj.setValue("REL", "status_code", 0)
-                        eObj.setValue("?", "recvd_initial_deposition_date", 0)
+            if not dataContainer.exists(cName):
+                if dataContainer.exists("entry") and dataContainer.exists("ma_data"):
+                    dObj = dataContainer.getObj("entry")
+                    entryId = dObj.getValue("id", 0)
+                    compModelSourceId = None
+                    if entryId.upper().startswith("MA_") or entryId.upper().startswith("MA-") or entryId.upper().startswith("AF_"):
```

Comment:
@piehld Do we need the if statement on line 633? If I understand correctly, lines 628 and 629 take care of the required conditionality. Am I missing something?